src/test/java/org/mockitousage/basicapi/ResetTest.java

```diff
@@ -63,6 +63,13 @@ public void shouldRemoveAllInteractions() throws Exception {
     }
 
     @Test
+    public void shouldRemoveAllInteractions2() throws Exception {
```

Comment:
I would rather not name it with `2`. Probably better as `should_remove_all_interactions_verify_no_interactions()`